core/finalize.go

```diff
@@ -7,15 +7,14 @@ import (
 	"strings"
 )
 
-// StorageConf represents a configuration for a container storage
+// Holds configuration for storage drivers
```

Comment:
no need for the "holds" part, something like "Configuration for storage drivers" would be better

Reply:
resolved by ddd1e3af7cc494fc31064fa68f5eec14a6b82c24